client/shipping-label/state/actions.js

```diff
@@ -236,6 +237,13 @@ export const confirmAddressSuggestion = ( group ) => ( dispatch, getState, { sto
 		expandFirstErroneousStep( dispatch, getState, storeOptions, group );
 	};
 
+	const errors = getFormErrors( getState(), storeOptions );
+
+	// Make sure packaging is error free before fetching new rates after address confirmation
+	if ( hasNonEmptyLeaves( errors.packages ) ) {
```

Comment:
You should check here than origin & destination are valid too. This function is the result of the user "confirming" a validated address, but the other one could still be invalid.